internal/pkg/term/color/color.go

```diff
@@ -71,3 +72,8 @@ func HighlightResource(s string) string {
 func HighlightCode(s string) string {
 	return HiCyan.Sprintf("`%s`", s)
 }
+
+// ProdColor colors the string to mark it is a prod environment.
+func ProdColor(s string) string {
```

Comment:
Can this be `Prod` instead of `ProdColor` so that it reads `color.Prod`?

Reply:
done!